app/containers/AccountInitializer/selectors.js

```diff
@@ -19,9 +19,27 @@ const makeSelectError = () =>
     substate.get('error'),
   );
 
+const makeSelectEosInit = () =>
```

Comment:
It is confusing to me when I see eosInit in the state. There is already eosService with eos state.
eosInit also contains current selected account details which I do not expect to see in eosInit based on name.
I recommend to keep eosService with all eos properties. And separate state for selectedAccount.